src/Uno.UI/UI/Xaml/DependencyPropertyValuePrecedences.cs

```diff
@@ -27,6 +27,11 @@ public enum DependencyPropertyValuePrecedences : int
 		/// </summary>
 		Local,
 
+		/// <summary>
+		/// Values set by a Popup to its child
+		/// </summary>
+		Popup,
```

Comment:
I think we can't include the PR as is in 4x, as this is a breaking change (as it changes a public enum and also moves all the subsequent members +1 in value. What do you think @Youssef1313 ?

Reply:
Yup it will be very risky to backport.

Reply:
Well, actually while public, I don't see how it can be used by external code, but I can't guarantee that 😄 

So let's stay on the safe side

Reply:
I don't think we should be adding a new precedence level for this particular scenario. It has a wide performance impact for a very specific use-case. Maybe there's another condition we can use to avoid this particular issue.